specificatie/publiek-rechtelijke-beperkingen.yaml

```diff
@@ -90,6 +90,11 @@ components:
           $ref: "domain.yaml#/components/schemas/NietNatuurlijkPersoonBeperkt"
         beperkingsgebied:
           $ref: "#/components/schemas/Beperkingsgebied"
+        beperktOnroerendeZaakIdentificaties:
```

Comment:
Voorstel voor naamgeving : beperkingOnroerendeZaakIdentificaties. Wij gebruiken op andere plekken (persoon_beperkt) de term beperkt in een andere context.